src/data/faq_de.json

```diff
@@ -606,8 +606,8 @@
                         "active": true,
                         "textblock": [
                             "So funktioniert die App:",
-                            "<ol><li>Alle Smartphones, die die App installiert haben, senden und empfangen pseudonymisierte Schlüssel, die dann lokal auf dem Smartphone gespeichert werden. Die Bluetooth-Verbindung ist nur für das Senden und Empfangen der pseudonymisierten Schlüssel notwendig.</li><li>Falls Sie positiv getestet wurden, können Sie Ihre Diagnoseschlüssel ('Positivkennung') auf den Server hochladen. Alle Smartphones mit der Corona-Warn-App empfangen in regelmäßigen Abständen die Schlüssel von diesem Server und prüfen lokal, ob sie einen übereinstimmenden Schlüssel via Bluetooth lokal gespeichert haben. Hierfür muss Ihr Smartphone Zugang zum Internet haben, zum Beispiel über WLAN oder mobile Daten.</li><li>Gibt es eine Übereinstimmung zwischen einem vom Server heruntergeladenen und einem lokal gespeicherten Schlüssel, dann bedeutet das, dass es eine Begegnung mit einer Person gab, die positiv auf Corona getestet wurde. Für diesen Abgleich von Übereinstimmungen benötigen Sie keine Bluetooth-Verbindung.</li></ol>",
-                            "Das <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md' target='_blank' rel='noopener noreferrer'>Solution-Architecture-Dokument</a> beschreibt den technischen Ablauf im Detail."
+                            "<ol><li>Alle Geräte, die die App installiert haben, senden und empfangen Zufalls-IDs, die dann lokal auf dem Gerät gespeichert werden. Die Bluetooth-Verbindung ist nur für das Senden und Empfangen der Zufalls-IDs notwendig.</li><li>Falls Sie positiv getestet wurden, können Sie Ihre über Bluetooth gesendeten Zufalls-IDs in das Corona-Warn-App Backend hochladen. Alle Geräte mit der Corona-Warn-App laden in regelmäßigen Abständen die Zufalls-IDs von diesem Backend herunter und prüfen lokal, ob sie einen übereinstimmenden Schlüssel via Bluetooth empfangen haben. Für das Hochladen und Herunterladen der Zufalls-IDs muss Ihr Gerät Zugang zum Internet haben, zum Beispiel über WLAN oder mobile Daten.</li><li>Gibt es eine Übereinstimmung zwischen einem vom Server heruntergeladenen und einem lokal gespeicherten Schlüssel, dann bedeutet das, dass Sie eine Begegnung mit einer Person hatten, die positiv auf Corona getestet wurde. Für diesen Abgleich von Übereinstimmungen benötigen Sie keine Bluetooth-Verbindung.</li><li>Auf Basis der gefundenen Risikobegegnungen errechnet die App Ihren persönlichen Risikostatus, welcher bestimmt, ob SIe ein \"niedriges Risiko\" (grüne Karte) oder \"hohes Risiko\" (rote Karte) angezeigt bekommen. Weitere Informationen zur Risikoberechnung finden sie im FAQ-Eintrag <a href='encounter_criteria' target='_blank' rel='noopener noreferrer'>Nach welchen Kriterien werden Begegnungen bewertet und wie unterscheiden sich die Handlungsempfehlungen?</a></li></ol>",

```

Comment:
The link to bookmark
a href='encounter_criteria' 
does not work. It should be
a href='#encounter_criteria'

Reply:
fixed